appdatabase/migrations/sql/1697123140_drop_include_watch_only_accounts.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings DROP COLUMN include_watch_only_account;
```

Comment:
@qfrank could you check if this causes issues on the iOS side?

Reply:
ios support `DROP COLUMN` now, so no more changes needed for this PR